src/components/Sidebar.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { updateCurrentChat } from '../actions';
+
+export const Sidebar = (props) => {
+  const { chats, updateCurrentChat, currentChat } = props;
+
+  const onClick = (i) => {
+    updateCurrentChat(i);
+  };
+
+  const renderChats = () => {
+    return chats.map((chat, i) => {
+      return (
+        <li
+          className={`sidebarItem ${currentChat === i ? 'selected' : null}`}
+          onClick={() => onClick(i)}
+          key={i}
```

Comment:
> We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny’s article for an in-depth explanation on the negative impacts of using an index as a key. If you choose not to assign an explicit key to list items then React will default to using indexes as keys.

https://reactjs.org/docs/lists-and-keys.html#keys